web/themes/custom/jcc_elevated/templates/paragraphs/paragraph--webform.html.twig

```diff
@@ -0,0 +1,28 @@
+{{ attach_library("jcc_storybook/Accordion") }}
+{{ attach_library("jcc_storybook/Container") }}
+{{ attach_library("jcc_storybook/Stack") }}
+
+{% set classes = [
+  'paragraph',
+  'paragraph__webform',
+  'box'
+] %}
+
+<div class="{{ classes|join(" ") }}">
+  <div class="container stack">
+
+    {% if content.field_lead or content.field_heading %}
```

Comment:
I'm seeing a pattern here...  Maybe this should just be a pattern on it's own.

Reply:
Removed this until a webform component can be created